plugins/imdb.py

```diff
@@ -116,18 +116,17 @@ def get_countries_and_languages(soup):
     else:
         lg_text = "No Languages Found!"
     if countries:
-        if len(countries) > 1:
-            ct_text = ', '.join([cnt["NAME"] for cnt in countries])
-        else:
-            ct_text = countries[0]["NAME"]
+        ct_text = countries["NAME"]
     else:
-        ct_text = "No Writer Found!"
+        ct_text = "No Country Found!"
     return ct_text, lg_text
 
 
 def get_credits_text(soup):
     pg = soup.get("sum_mary")
-    direc = pg.get("Director")
+    direc = pg.get("Directors")
+    if not direc:
+        direc = pg.get("Director")
```

Comment:
😳🤔😔

Reply:
Your api returing sum_mary has key Director if movie has only 1 Director else key is Directors

Reply:
![IMG_20210625_090327](https://user-images.githubusercontent.com/65902764/123365665-50a81180-d594-11eb-9bf4-2d2f63db41fb.jpg)
![IMG_20210625_090349](https://user-images.githubusercontent.com/65902764/123365668-530a6b80-d594-11eb-94b5-bb4449279094.jpg)